src/utils/logic-app/templateUtils.ts

```diff
@@ -64,21 +65,37 @@ export function generateDeploymentTemplateParameters(parameters: Record<string,
 }
 
 export function generateParameters(workflow: Workflow) {
+    let parameters =  Object.assign({}, workflow.definition.parameters);
+
+    for (var key in parameters) {
+        // Rename parameters by appending `workflows_{workflow}` in front to avoid conflict of the same parameter name between multiple Logic Apps
+        parameters[`workflows_${workflow.name}_parameters_${key}`] = {value: parameters[key].defaultValue};
+        delete parameters[key];
+    }
+
     return {
         $schema: "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
         contentVersion: "1.0.0.0",
         parameters: {
-            ...generateTemplateParameter(workflow.name!)
+            ...generateTemplateParameter(workflow.name!), ...parameters
         }
     };
 }
 
 export function generateTemplate(workflow: Workflow) {
+    let parameters =  Object.assign({}, workflow.definition.parameters);
+
+    for (var key in parameters) {
```

Comment:
`for (const key of Object.keys(parameters)) {`

Reply:
Fixed pending update